packages/messenger-widget/src/components/Preferences/Notification/bl.ts

```diff
@@ -0,0 +1,187 @@
+import { log } from '@dm3-org/dm3-lib-shared';
+import { IVerificationModal } from './VerificationModal';
+
+export const BTN_CLASS =
+    'verify-btn font-weight-400 font-size-12 border-radius-4 line-height-24';
+
+export const INPUT_FIELD_CLASS =
+    'notification-input-field font-weight-400 font-size-14 border-radius-6 w-100 line-height-24';
+
+export enum VerificationMethod {
+    Email = 'Email',
+    Telephone = 'Telephone',
+}
+
+export const getVerficationModalContent = (
```

Comment:
Instead of keeping every function in  bl.ts please move the Logic towards its hook that encapsulates everything related to Notifications, That hook should expose an interface for every functionality required by the Components in the Notification directory. 
The logic of the different methods ie mail, push notification etc. can be moved to their own functions that can be tested independently 

hooks
-Notifications
--useNotifications // provides an interface to all methods
--Methods
---email
---sms
---push
---...